Split Importing content assembly (#4774)
theforeman/foreman-documentation: guides/common/modules/con_authenticating-and-securing-content-sources.adoc

:_mod-docs-content-type: CONCEPT

> [id="authenticating-and-securing-content-sources_{context}"]

Reviewer: ```suggestion
[id="authenticating-and-securing-content-sources"]
```

If possible, I suggest to drop the trailing `_{context}` in the anchor.

> = Authenticating and securing content sources

Reviewer 2: This module is really about content credentials, so it makes more sense if the title reflects that.

```suggestion
[id="authenticating-and-securing-content-with-content-credentials_{context}"]
= Authenticating and securing content with content credentials
```

[role="_abstract"]
Remote content sources often use HTTPS and signed metadata or packages.
Import SSL certificates when upstream servers use custom or private certificate authorities so {Project} can connect securely.

> Import GPG keys so {Project} and managed hosts can verify signatures on RPM or Deb content.

Reviewer: This is different for Yum vs Deb content:

Yum: hosts check the signature of RPM packages; optional: signed metadata though uncommon to my understanding.
Deb: Foreman+Katello checks the signature of metadata; Pulp signs the metadata (=`Release` files); hosts need the `pulp_deb_signing.key` to verify the metadata from Foreman+Katello

```suggestion
ifdef::katello[]
Import GPG keys so {Project} and registered hosts can verify signatures on RPM packages or metadata of Deb content.
endif::[]
ifdef::orcharhino[]
ifdef::almalinux,amazon_linux,centos,oracle_linux,red_hat_enterprise_linux,rocky_linux,suse_linux_enterprise_server[]
Import GPG keys so registered hosts can verify signatures on Yum content when they consume content from {Project}.
endif::[]
ifdef::debian,ubuntu[]
Import GPG keys so {Project} can verify signatures on metadata of Deb content.
endif::[]
endif::[]
ifdef::satellite[]
Import GPG keys so registered hosts can verify signatures on Yum content when they consume content from {Project}.
endif::[]
```
